Zend/zend_fibers.stub.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/** @generate-class-entries */
+
+final class Fiber
```

Comment:
May want to add `/** @strict-properties */` to forbid adding properties on the fiber.